### PR TITLE
Ensure JIT Docs use correct casing

### LINKED ISSUE #2024: Ensure JIT Docs use correct casing
See: https://github.com/Microsoft/dotnet/pull/386

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #2024 

cc/ @richlander @BruceForstall

### CODE REVIEW

**Comment by BruceForstall** [comment]
LGTM. Thanks!

**Review by BruceForstall — APPROVED** [review]

**Comment by GuardRex** [comment]
We'll be adding some topics to this shortly.

**Review by rpetrusha — APPROVED** [review]
Thanks, @GuardRex. You included some topics that I'd missed on MSDN.

File: docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-6.md [hunk 1]
```diff
@@ -98,4 +98,4 @@ In rare cases, runtime changes may affect existing apps that target the previous
...
\ No newline at end of file
+|Exception handling (returning from a `try` region)|Unlike the older JIT64 just-in-time compiler, the new 64-bit JIT compiler does not allow an IL `ret` instruction in a `try` region.|Returning from a `try` region is disallowed by the ECMA-335 specification, and no known managed compiler generates such IL. However, the JIT64 compiler will execute such IL if it is generated using reflection emit.<br /><br /> If your app generates IL that includes a `ret` opcode in a `try` region, you can:<br /><br /> -   Target the .NET Framework 4.5 or add the [\<useLegacyJit>](../../../docs/framework/configure-apps/file-schema/runtime/uselegacyjit-element.md) element to your application configuration file use the old JIT compiler and avoid the change.<br />-   Update the generated IL to return after the `try` region.<br />-|Edge|
```
Comment (rpetrusha) [inline docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-6.md]: One change: "use the old JIT compiler and avoid..." -> "to use the old JIT compiler and avoid..."

**Review by mairaw — APPROVED** [review]
Thank you @GuardRex. Merging this now.

**Pull request merged**